### ISSUE TITLE
Problem in SUPERB benchmark reproducibility and superb artifacts server

### ISSUE THREAD

**Issue by amitmeghanani** [s3prl/s3prl#498]
# Problems in reproducing the same numbers with default downstream configs.

Are the default downstream configs are different than the one used in SUPERB?
I tried to access it via https://github.com/s3prl/s3prl/blob/main/s3prl/downstream/docs/superb_artifacts.md but the artifacts  server is not working.
Is to normal to get different results than the reported benchmark with default configs? For example for WavLM Phone recognition task, the reported PER is 4.84 but I am getting 5.5.

**Comment by leo19941227** [comment]
Hi, this is normal, you need to tune the learning rate, and we tune from 1.0e-1 to 1.0e-7 with log scale.
I am relaunching the SUPERB server.

**Comment by leo19941227** [comment]
Here is my PR results on WavLM Large
<img width="327" alt="截圖 2023-09-17 上午1 58 36" src="https://github.com/s3prl/s3prl/assets/33196053/69a9c6b7-82cc-4d0e-bfe9-029ac26982cb">

**Comment by leo19941227** [comment]
Or you can refer to the WavLM paper's Appendix for the fine-tuning configs:
<img width="1058" alt="截圖 2023-09-17 上午2 00 40" src="https://github.com/s3prl/s3prl/assets/33196053/f85f48d1-179a-4c44-8d06-1b8d3a86635e">

**Comment by leo19941227** [comment]
Despite they use large batch size, but in my experience learning rate has much more impact on the final results than the batch size. Larger batch size get slightly better results, but using smaller batch size and tuning LR properly can still give you good results.

**Comment by amitmeghanani** [comment]
Hello,
Thank you very much for the clarification and  relaunching the SUPERB server.
